packages/cactus-plugin-htlc-eth-besu-erc20/src/main/typescript/generated/openapi/typescript-axios/.npmignore

```diff
@@ -0,0 +1 @@
+# empty npmignore to ensure all required files (e.g., in the dist folder) are published by npm
```

Comment:
@jordigiam Please ignore this file in the openapi generator ignore file + make sure this file is deleted from the change

Reply:
Done